modules/ROOT/pages/getting-started/toolbox.adoc

```diff
@@ -12,58 +12,63 @@ With it, you can:
 . Build the Neo4j GraphQL schema.
 . Experiment, query, and play.
 
+
 == Connect to a Neo4j database
 
 Before you start using the Toolbox, make sure you have followed all xref:index.adoc#_requirements[requirements] to run a Neo4j database.
 You can use https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] or https://neo4j.com/docs/aura/auradb/[Neo4j AuraDB] for that.
 
+
 == Set the type definitions
 
-. Set your type definitions directly in the Toolbox editor or introspect the Neo4j database you connected to.
-+
-If you followed the xref:getting-started/index.adoc[Getting started tutorial], you should see these type definitions in the GraphQL Toolbox:
-+
+Set your type definitions directly in the Toolbox editor or introspect the Neo4j database you connected to.
+
+If you followed the xref:getting-started/graphql-self-hosted.adoc[self-hosted GraphQL tutorial], use **Introspect** to see these type definitions in the GraphQL Toolbox:
```

Comment:
It's worth bearing in mind that if the user has done extra things in their db since that tutorial, they will likely see different type definitions when they introspect the db